server/src/main/java/com/cloud/network/Ipv6ServiceImpl.java

```diff
@@ -167,6 +167,11 @@ private boolean isPublicIpv6PlaceholderNic(NicVO nic) {
         return new Pair<>(nic.getIPv6Address(), vlanOptional.get());
     }
 
+    private void publishPublicIpv6AssignActionEvent(final Network network, final String ipv6Address) {
+        final String description = String.format("Assigned public IPv6 address: %1$s for network ID: %2$s. Static route: %3$s via %1$s", ipv6Address,  network.getUuid(), network.getIp6Cidr());
```

Comment:
Do we need to mention 'static route' here?

Maybe we can just leave that piece out of rename it that the output looks like:

```
prefix=2001:db8:100::/64 next-hop=2001:db8::80
```